go/log-stressor/log-stressor.go

```diff
@@ -96,7 +146,7 @@ func NewDistributionProfile(payloadGen, distribution string) DistributionProfile
 	profile := payloadGen + "/" + distribution
 	switch profile {
 	case "fixed/fixed":
```

Comment:
@ajaygupta978 can you change this payloadGen to be Finite and not fixed so the case will be for finite /fixed and not as it is now.

Reply:
Also, why is this a `switch` ... did you had in mind that you need more options / cases here ???

Reply:
@eranra This method was not written by me so I am not aware about details.